src/sys/linux/random.cr

```diff
@@ -30,6 +30,21 @@ module Sys
       end
     end
 
+    # Returns a random UInt32.
+    def self.random_number : UInt32
```

Comment:
I notice `Random.rand_type` already exists. I'll use it, thought I would prefer the `rand_bytes` naming, and what about making it public?